game/randomGen.h

```diff
@@ -0,0 +1,17 @@
+#ifndef RANDOM_GEN_H_
+#define RANDOM_GEN_H_
+
+#include <random>
+
+class RandomGenerator {
+    const unsigned int min;
+    const unsigned int max;
+    static std::random_device rd;
```

Comment:
I believe you need to initialize the random device in the implementation file. move.cc on the action-implementation branch has an example if you want.